packages/tables/resources/views/components/defer-loading-indicator.php

```diff
@@ -0,0 +1,13 @@
+<div {{ $attributes->class([
+    'filament-tables-empty-state flex flex-1 flex-col items-center justify-center p-6 mx-auto space-y-6 text-center bg-white',
+    'dark:bg-gray-800' => config('tables.dark_mode'),
+]) }}>
+    <div @class([
+        'flex items-center justify-center w-16 h-16 text-primary-500 rounded-full bg-primary-50',
```

Comment:
Dark background color should be handled.

Reply:
Looks good. Sorry, I missed the separate dark mode class the line below. In v3 they're all on a single line. 🤦‍♂️